doc/Contribute.md

```diff
@@ -100,6 +101,27 @@ associated documentation, if we have any.
 |__top__/rules                                                          |Supporting bazel definitions.                          |
 |[__top__/testing](tests.md)                                            |Bazel targets to run CF smoke and acceptance tests.    |
 
+
+## Updating subcharts
+
+The kubecf helm chart includes a number of subcharts. They are declared in
+[requirements.yaml](../deploy/helm/kubecf/requirements.yaml). For the
+convenience of development they are included in unpacked form directly in
+this repo, so version changes can be inspected with regular `git` tools,
+and the subcharts can be searched with `grep` etc.
+
+The procedure to update the version of a subchart is:
```

Comment:
Can we not have a `bazel` target for this to ensure we don't screw this up when we copy & paste? (Yes, we want to get rid of bazel, but until the replacement _starts_ to exist it's still our build system.)

Reply:
I agree that we should, but personally I'm unable to create a `bazel` target. I can create a shell script that you can run from the `bazel` target, but that is practically already part of the docs I wrote.

So, we can:

1. Have somebody else contribute the `bazel` target for this PR, or
2. Delay merging this PR until we have switched build systems, and then add the shell script, or
3. Be careful for the next month or two to make sure we follow the docs when bumping subcharts.

Personally I find (3) acceptable, given that all PRs go through code review, but I'm fine with the other options too.

The advantage of merging this earlier is that we will already have diffs available for any subchart bumps we do before switching build systems.

Reply:
I think we _do_ want at least a shell script; even if we don't run it from bazel, we want to make sure we don't make dumb typos / copy & paste errors (I always assume I can find ways to mess up).

We can tack on the bazel target later.

Reply:
Script added in 81541c2